ztools/x509/extensions_test.go

```diff
@@ -48,7 +51,7 @@ func (s *ExtensionsSuite) SetUpTest(c *C) {
 
 func (s *ExtensionsSuite) TestEncodeDecodeSAN(c *C) {
 	for _, cert := range s.parsedCerts {
-		if (cert.Issuer.CommonName != "SAN Test"){
+		if cert.Issuer.CommonName != "SAN Test" {
```

Comment:
in the SetUpTest functions, all certificates are parsed in one go and stored into ExtensionsSuite struct. If we were to load the certificate in the test, it will get parsed a second time.